octree/basic.go

```diff
@@ -131,8 +131,13 @@ func (octree *basicOctree) At(x, y, z float64) (pc.Data, bool) {
 	return nil, false
 }
 
-// Iterate TODO: Implement iterate for octree.
+// Iterate takes in the number of batches, a batch id and a function to apply to each point in a basic octree
+// until false is returned. These parameters are passing them to iterateRecursive along with the starting index
+// (0) to iterate though a basic octree if the given number of batches is greater than 0.
 func (octree *basicOctree) Iterate(numBatches, myBatch int, fn func(p r3.Vector, d pc.Data) bool) {
```

Comment:
@bhaney any issue with deviating from pointcloud's `Iterate` by implementing `currentBatch`? (also if you'd like I can also change this in the pointcloud implementation)

Reply:
You can just name it differently here in your function, no need to change the name elsewhere.
As long as the signature is `Iterate(int, int, func(r3.Vector, pc.Data) bool)` it will work